.gitignore

```diff
@@ -0,0 +1 @@
+/opencv_build/
```

Comment:
This icon looks scary, I would add newlines to make github happy 😃

Reply:
Done